src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/AuthorMap/AuthorMap.xaml.cs

```diff
@@ -63,22 +53,30 @@ public AuthorMap()
             InitializeComponent();
 
             // Create the UI, setup the control references and execute initialization
-            Initialize();
+            _ = Initialize();
         }
 
-        private void Initialize()
+        private async Task Initialize()
         {
-            // Create the map
-            MyMapView.Map = new Map();
+            ArcGISLoginPrompt.SetChallengeHandler();
+
+            bool loggedIn = await ArcGISLoginPrompt.EnsureAGOLCredentialAsync();
+
+            // Show a plain gray map in the map view
+            if (loggedIn)
+            {
+                MyMapView.Map = new Map(BasemapStyle.ArcGISLightGray);
+                _ = MyMapView.Map.Basemap.LoadAsync();
```

Comment:
Is loading the basemap specifically the same pattern used in the AuthorMap samples on the other .NET platforms?